packages/devtools-core/tests/actions/index.test.ts

```diff
@@ -31,4 +32,14 @@ describe("actions", () => {
     expect(state.app.actionLog.length).toBe(1);
     expect(state.app.actionLog[0].actionName).toBe(testActionLog[0].actionName);
   });
+
+  it("clears action and render lgos", async () => {
+    const { store } = model.createStore({
+      initState: populatedState,
+    });
+
+    const { state } = await store.dispatch(clearLogs)();
```

Comment:
Maybe assert that the action log length is > 0 initially, just as a sanity check